game/state/rules/battle/battlemap.cpp

```diff
@@ -134,10 +135,12 @@ sp<Battle> BattleMap::createBattle(GameState &state, StateRef<Organisation> oppo
 	}
 	auto alienOrg = state.getAliens();
 
+	float hostilesMultiplier = config().getFloat("OpenApoc.Cheat.HostilesMultiplier");
+
 	int countAliens = 0;
 	for (auto &a : *aliens)
 	{
-		for (int i = 0; i < a.second; i++)
+		for (int i = 0; i < (int)(a.second * hostilesMultiplier); i++)
```

Comment:
Try to avoid c-style casts - preference is for static_cast<int>(whatever) (or an actual rounding function)

Reply:
I changed these to use std::round, which is more appropriate anyways